README.md

```diff
@@ -214,7 +214,14 @@ AWS Lambda lets you run code without provisioning or managing servers. You pay o
    * Provide a skill name that represents the new skill you are creating.
    * Come up with a cool Invocation Name that users will use to invoke your skill.  Ensure that the invocation name you choose stays clear of pitfalls listed in the table below. Column on the right provides examples of invocation names that will definitely fail certification.
 
-   ![](https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/tutorials/general/invocation_name_requirements._TTH_.png)
+   Invocation name requirements | Examples of incorrect invocation names
+   ---------------------------- | --------------------------------------
+   The skill invocation name must not infringe upon the intellectual property rights of an entity or person. | korean air; septa check
+   Invocation names should be more than one word (unless it is a brand or intellectual property), and must not be names or places. | Horoscope; trivia; guide; New York
+   Two word invocation names are not allowed when one of the words is a definite article, indefinite article or a preposition. | any poet; the bookie; the fool
```

Comment:
I took the liberty of adding a period after "preposition". Besides that this should be the exact text as before.